src/Phinx/Console/Command/Create.php

```diff
@@ -112,6 +177,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // Do we need to do the post creation call to the creation class?
+        if (!empty($creationClassName)) {
```

Comment:
Yeah, but if you have a separate class responsible of creating the template file, you could add your custom post-process logic there. No need to for that into phinx api.